index.html

```diff
@@ -353,6 +354,69 @@ <h2>Ballerina in Action</h2>
                   </div>
                </div>
             </div>
+            <div id="fifthTab" class="tab-pane fade"  >
+            <div class="row cdisplayFlex" style="background-color: #f5f5f5;">
+                <div class="col-lg-12 col-md-12 col-sm-12 focusPane" style="padding:20px;border: 1px #c7c7c7 solid;">
+                    <div class="FullCode" style="background-color: #f5f5f5;">
+                        <div class="highlight">
+                            <pre class="language-ballerina basic ballerina">
+                                <code class="language-ballerina ballerina" style="font-size: 12px !important;">
+import ballerinax/kafka;
```

Comment:
Let's swap the order of these imports? I would use `Optimize imports` action in our VSCode plugin

Reply:
Added in 76f775c